frontend/src/components/common/Resource/PortForward.tsx

```diff
@@ -11,6 +11,7 @@ import {
   startPortForward,
   stopOrDeletePortForward,
 } from '../../../lib/k8s/apiProxy';
+import { PortForward as PortForwardState } from '../../../lib/k8s/apiProxy/portForward';
```

Comment:
It would be good to expand on the commit message explaining in more detail what's happening.

Reply:
good idea. added a better description to the commit